### PR TITLE
Add Sol Index

### PR DESCRIPTION
**Pull request by rikisann**

### CODE REVIEW

**Review by 0xYankee-Raccoons — APPROVED** [review]

File: assets/transparent sol.png:Zone.Identifier [hunk 1]
```diff
@@ -1,3 +0,0 @@
-[ZoneTransfer]
-ZoneId=3
-HostUrl=https://www.remove.bg/
```
Comment (MrPiNkY2022) [inline assets/transparent sol.png:Zone.Identifier]: Remove file

**Review by MrPiNkY2022 — COMMENTED** [review]
Check code and remove assets/transparent sol.png:Zone.Identifier file

**Review by MrPiNkY2022 — COMMENTED** [review]
Good job

**Review by MrPiNkY2022 — COMMENTED** [review]
Good job

**Pull request merged**